zeppelin-server/src/main/java/org/apache/zeppelin/realm/ActiveDirectoryGroupRealm.java

```
@@ -152,7 +152,7 @@ private String getSystemPassword() {
           password = new String(credEntry.getCredential());
         }
       } catch (Exception e) {
-        log.debug("ignored exception: " + e.getMessage());
+        log.debug("", e);
```

eh, I meant `log.debug("ignored exception", e);`

---

or better, `log.debug("ignored error from getting credential entry from provider", e);`
or `log.debug("ignored error from getting credential entry from keystore", e);`

---

My fault. I added a new commit addressing your comment.